test/bases/renku_data_services/data_api/test_projects.py

```diff
@@ -94,6 +94,17 @@ async def test_project_creation(sanic_client, user_headers, regular_user, app_co
         "http://renkulab.io/repository-2",
     }
 
+    # same as above, but using namespace/slug to retreive the pr
+    _, response = await sanic_client.get(
+        f"/api/data/projects/{payload['namespace'].lower()}/{payload['slug']}", headers=user_headers
```

Comment:
We can remove `.lower()` here if we do the lowercase conversion in the `get_project_by_namespace_slug` function.

Reply:
Fixed in 651ceb960de0d5064cd9c283d7f01d0c1e56c02c